frontend/test/metabase/scenarios/models/models-with-aggregation-and-breakout.cy.spec.js

```diff
@@ -0,0 +1,48 @@
+import {
+  restore,
+  modal,
+  popover,
+  visualize,
+  openOrdersTable,
+} from "__support__/e2e/helpers";
+
+import { turnIntoModel } from "./helpers/e2e-models-helpers";
+
+describe("scenarios > models with aggregation and breakout", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    cy.intercept("POST", "/api/dataset").as("dataset");
+    cy.intercept("PUT", "/api/card/*").as("updateCard");
+  });
+
+  it("should be possible to convert a question with an aggregation and breakout into a model", () => {
+    openOrdersTable({ mode: "notebook" });
+
+    // Add an aggregation
+    cy.findByText("Summarize").click();
+    cy.findByText("Number of distinct values of ...").click();
+    cy.findByText("Product ID").click();
+
+    // Add a breakout
+    cy.findByText("Pick a column to group by").click();
+    popover().within(() => {
+      cy.findByText("Created At").click();
+    });
```

Comment:
Is it necessary for the test that a mode is created via Metabase UI? Creating a question with `cy.request("POST", "/api/card")` would make the test run much faster and make it more reliable overall

Reply:
sure, updated to create the question not through the ui